api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -28,7 +32,7 @@
 @Mapper(
     componentModel = "spring",
     collectionMappingStrategy = CollectionMappingStrategy.TARGET_IMMUTABLE,
-    uses = {CommonMappers.class})
+    uses = {CommonMappers.class, CohortMapper.class, CohortReviewMapper.class, ConceptSetMapper.class, DataSetMapper.class})
```

Comment:
Please list these one per line like we do in import statements in the test classes.